WebDriverAgentLib/Commands/FBSessionCommands.m

```diff
@@ -162,6 +165,11 @@ + (NSArray *)routes
   return FBResponseWithObject(@(state));
 }
 
++ (id<FBResponsePayload>)handleActiveAppsList:(FBRouteRequest *)request
+{
+  return FBResponseWithObject([XCUIApplication fb_activeAppsInfo]);
```

Comment:
WDA has an endpoint to return appinfo, bundleid, process id, name and environment variables. It has not been released yet. So, what do you think of modifying here like below and remove it?

https://github.com/appium/WebDriverAgent/blob/master/WebDriverAgentLib/Commands/FBCustomCommands.m#L60

```objc
+ (id<FBResponsePayload>)handleActiveAppsList:(FBRouteRequest *)request
{
  NSMutableArray<NSDictionary<NSString *, id> *> *result = [[NSMutableArray alloc] init];
  NSArray *appsInfo = [XCUIApplication fb_activeAppsInfo];
  for (NSDictionary *appInfo in appsInfo) {
    FBApplication *app = [FBApplication fb_activeApplicationWithDefaultBundleId:appInfo[@"bundleId"]];
    NSDictionary *processArguments = @{};
    if (app != nil) {
      processArguments = @{
        @"args": app.launchArguments,
        @"env": app.launchEnvironment
      };
    }

    [result addObject:@{
      @"pid": appInfo[@"pid"] ?: @"",
      @"bundleId": appInfo[@"bundleId"] ?: @"",
      @"name": app.identifier ?: @"",
      @"processArguments": processArguments,
    }];
  }
  return FBResponseWithObject(result);
}
```

Reply:
I wanted to reduce/list endpoint to a minimum and avoid extra calls to the accessibility if possible. Creation of FBApplication instance is already quite expensive as it touches the internal process monitoring machinery. Lets keep it at minimum for now.

Reply:
👍 make sense